src/vcpkg/base/system.cpp

```diff
@@ -11,6 +11,12 @@
 #include <sys/sysctl.h>
 #endif
 
+#if defined(_WIN32)
+#include <processenv.h>
+#else
+extern char** environ;
+#endif
```

Comment:
```suggestion
#if !defined(_WIN32)
extern char** environ;
#endif
```

Reply:
We need `#include <processenv.h>` for mingw

Reply:
Hmmm can you add a comment to that effect then?